GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2733,16 +2735,15 @@ private void DiffContextMenu_Opening(object sender, System.ComponentModel.Cancel
                 artificialRevSelected = artificialRevSelected || selectedRevisions[selectedRevisions.Count - 1].IsArtificial();
 
             // disable items that need exactly one selected item
-            bool isExcactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
-            var isCombinedDiff = isExcactlyOneItemSelected &&
+            bool isExactlyOneItemSelected = DiffFiles.SelectedItems.Count() == 1;
+            var isCombinedDiff = isExactlyOneItemSelected &&
                 DiffFiles.CombinedDiff.Text == DiffFiles.SelectedItemParent;
-            var enabled = isExcactlyOneItemSelected && !isCombinedDiff;
```

Comment:
Okay, I made the exact change I mentioned above:
```
openWithDifftoolToolStripMenuItem.Enabled = !isCombinedDiff;
```
This corresponds with what you said: disable difftool menu if right-clicking on a combined diff file, otherwise we enable it. I've tested it, and it works fine now. Along the way, I learned what a "combined diff" is :)

Thanks for seeing this problem!